open-sphere-plugins/my-places/src/main/java/io/opensphere/myplaces/controllers/RoiContextMenuProvider.java

```diff
@@ -0,0 +1,64 @@
+package io.opensphere.myplaces.controllers;
+
+import java.util.List;
+
+import javax.swing.JMenuItem;
+
+import de.micromata.opengis.kml.v_2_2_0.Placemark;
+import io.opensphere.core.Toolbox;
+import io.opensphere.core.control.action.ContextMenuProvider;
+import io.opensphere.core.geometry.PolygonGeometry;
+import io.opensphere.mantle.data.DataGroupInfo.DataGroupContextKey;
+import io.opensphere.mantle.util.MantleToolboxUtils;
+import io.opensphere.myplaces.models.MyPlacesDataTypeInfo;
+import io.opensphere.myplaces.specific.regions.utils.RegionUtils;
+
+/**
+ * Provides the context menu for ROI selections.
+ */
+public class RoiContextMenuProvider implements ContextMenuProvider<DataGroupContextKey>
+{
+
+    /** The toolbox. */
+    private final Toolbox myToolbox;
+
+    /** The Current placemark. */
+    private Placemark myPlacemark;
```

Comment:
I'm guessing you don't actually need to store myPlacemark and myType as members?